holoviews/streams.py

```diff
@@ -162,15 +163,15 @@ def reset(self):
                     setattr(self, k, p.default)
 
 
-    def add_subscriber(self, subscriber):
+    def add_subscriber(self, subscriber, precedence=0):
         """
         Register a callable subscriber to this stream which will be
         invoked either when event is called or when this stream is
         passed to the trigger classmethod.
         """
         if not callable(subscriber):
             raise TypeError('Subscriber must be a callable.')
-        self._subscribers.append(subscriber)
+        self._subscribers.append((precedence, subscriber))
```

Comment:
Right, you would need to use a key function in the sort. I realized this before I saw your comment...